azurefunctions-extensions-base/azurefunctions/extensions/base/meta.py

```diff
@@ -87,10 +87,34 @@ def get_raw_bindings(cls, indexed_function, input_types):
         return utils.get_raw_bindings(indexed_function, input_types)
 
     @classmethod
-    def check_supported_type(cls, subclass: type) -> bool:
-        if subclass is not None and inspect.isclass(subclass):
-            return issubclass(subclass, sdkType.SdkType)
-        return False
+    def check_supported_type(cls, annotation: type) -> bool:
+        if annotation is None:
+            return False
+
+        # The annotation is a class/type (not an object) - not iterable
+        if (isinstance(annotation, type)
+                and issubclass(annotation, sdkType.SdkType)):
+            return True
+
+        # An iterable who only has one inner type and is a subclass of SdkType
+        return cls.__is_iterable_supported_type(annotation)
+
+    @classmethod
+    def __is_iterable_supported_type(cls, annotation: type) -> bool:
```

Comment:
Why is this method prefixed with  __? Private methods are prefixed with a single underscore